R/data_remove.R

```diff
@@ -1,10 +1,10 @@
-#' @rdname data_reorder
+#' @rdname data_relocate
 #' @examples
 #' # Remove columns
 #' head(data_remove(iris, "Sepal.Length"))
 #' @export
 data_remove <- function(data, pattern, ...) {
-  new <- data[!names(data) %in% pattern]
+  new <- data[!grepl(pattern, names(data))]
```

Comment:
I think this makes more sense, but not sure where this function is used in our packages? It might break code. We could also add an argument like `regex`  or `fixed`. to switch whether `pattern` is a regular expression or a fixed string.

@DominiqueMakowski @IndrajeetPatil ?

Reply:
> not sure where this function is used in our packages?

In multiple places:
https://github.com/search?q=org%3Aeasystats+data_remove&type=code

Not sure how this change affects them.

Reply:
ok, we need to take into account multiple patterns.